.github/workflows/dbt_slim_ci.yml

```diff
@@ -68,6 +68,16 @@ jobs:
       - name: dbt test initial model(s)
         run: "dbt test $PROFILE --select state:new state:modified --exclude tag:prod_exclude tag:remove --defer --state . --project-dir ${PROJECT_DIR}"
 
+      - name: dbt test dex trades specific model(s)
+        shell: bash
+        run: |
+          if git diff --name-only ${{ github.event.pull_request.base.sha }}..${{ github.event.pull_request.head.sha }} models/_sector/dex/trades/ | grep -q "models/_sector/dex/trades/"; then
+            dbt run --models dex_base_trades
+            dbt run --models dex_trades
+            dbt test --models test_check_null_amount_usd
+          fi
+        working-directory: ${{env.PROJECT_DIR}}
+
```

Comment:
Hi @jeff-dude, I think I may need some helps for this part.

The test seems working now.